addons/sourcemod/scripting/mce/cvars.inc

```diff
@@ -202,7 +210,11 @@ stock void CvarsInit()
 	g_Cvar_TimerUnlockNoms = CreateConVar("mce_locknominations_timer", "15.0", "Unlock nominations after a vote. Time in seconds.", _, true, 0.0, true, 60.0);
 	g_Cvar_ShowNominator = CreateConVar("mce_shownominator", "1", "See who nominated the map which won the vote", _, true, 0.0, true, 1.0);
 	g_Cvar_SharedCDMode = CreateConVar("mce_sharedcd_mode", "1", "Group maps with shared cooldowns mode (See ExcludeMap native docs for mode)", _, true, 0.0, true, 2.0);
-
+	g_Cvar_UseTiers = CreateConVar("mce_use_tiers", "1", "Use tiers for maps (map difficulty)", _, true, 0.0, true, 1.0);
```

Comment:
I personally don't find a need for having a dedicated convar for enabling map tiers IMO. I think it should be where if the plugin detects tiers being set then it should automatically enable tier system. Only reason why I would see a cvar being in use is if you want to use map tiers for nominating maps (e.g. you type `!nominate` and the menu is sorted by tiers that players can select.